### PR TITLE
QUEST_JOBSTEP.tsv Review Lines Part 1

### PR DESCRIPTION
**Pull request by ttgmichael**
Reviewing lines for Jobstep to check for outdated terms used, grammar problems, typos, and weird item references. 

Let me know if you catch any small things within these changes.

### CODE REVIEW

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -46,97 +46,97 @@ QUEST_JOBSTEP_20150317_000045	{memo X}Now go to Gintara Highway at Spine Heart D
...
 QUEST_JOBSTEP_20150317_000135	{memo X}As compensation for seeking cooperation, just support 5 boards of wooden buckler type.{nl}Defeating Upents at Indigo Trees Plateau will be enough.
-QUEST_JOBSTEP_20150317_000136	Thank you for the supplies, but the Highlander Master wouldn't have sent you for nothing. {nl}Tell me what you want. {nl}
-QUEST_JOBSTEP_20150317_000137	An overall cooperation between the Masters. Alright. {nl}Tell him that the Peltastas are for it. 
+QUEST_JOBSTEP_20150317_000136	$Thank you for the supplies, but the Highlander Master wouldn't have sent you for just that. {nl}Tell me what you want. {nl}
+QUEST_JOBSTEP_20150317_000137	$An overall cooperation between the Masters? Certainly. {nl}Tell him that the Peltastas will cooperate. 
 QUEST_JOBSTEP_20150317_000138	{memo X}If the Masters unite, it won't be long until the kingdom rises again. {nl}You did a good job. I will teach you my swordsmanship.
-QUEST_JOBSTEP_20150317_000139	That is enough. {nl}Get the Peltasta's cooperation. 
+QUEST_JOBSTEP_20150317_000139	$That should be enough. {nl}Now go seek the cooperation with the Peltastas.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: with the Peltastas. -> with the Peltasta Master.

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -261,27 +261,27 @@ QUEST_JOBSTEP_20150317_000260	There are still many people who need treatment. {n
 QUEST_JOBSTEP_20150317_000261	The Goddess helps those who help. {nl}It may not be much, but I hope I can be of assistance in taking you to the next level.
 QUEST_JOBSTEP_20150317_000262	{memo X}Cataphract...{nl}I'm not sure if you have the strength to learn.
 QUEST_JOBSTEP_20150317_000263	{memo X}I like your courage.{nl}Show your skills by defeating Sparnasman in the Rhombuspaving Dale. 
-QUEST_JOBSTEP_20150317_000264	Great work. You seem qualified to learn my skills. {nl}Ah. I apologize for underestimating you.
+QUEST_JOBSTEP_20150317_000264	Great work. You seem qualified to learn my skills. {nl}Oh. and I apologize for underestimating you.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Oh. and I apologize for -> Oh, and I apologize for

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**